stories/Icon/ExampleIcons.jsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/no-array-index-key */
+import styled from 'styled-components';
+import Icon from '../../components/Icon';
+import { colors } from '../../components/shared/theme';
+
+const IconWrapper = styled.div`
+  padding: 10px;
+  display: inline-block;
+`;
+
+const exampleIcons = [
+  <Icon name="info" size="small" />,
```

Comment:
The default color is loading in black.
Probably the our new doc is not using Quantum Typography.
In this component, its sets the gray color on Body.

New:
![image](https://user-images.githubusercontent.com/3389749/180290273-ac90c3fc-ccf2-4d11-9e00-dedf645f0682.png)

earlier:
![image](https://user-images.githubusercontent.com/3389749/180290508-3938ac92-824c-49ad-9294-6e70e22d07bc.png)

Reply:
I think it's more interesting to open a task to check this case